server/service/src/sync/test/mod.rs

```diff
@@ -337,9 +337,6 @@ pub(crate) async fn check_records_against_database(
             UserPermission => {
                 check_delete_record_by_id!(UserPermissionRowRepository, con, id)
             }
-            Name => {
```

Comment:
Removing sync deletes for names since names should only be soft deleted... will need to implement soft deletes for name in both om and mSupply....

Reply:
mSupply is likely just going to continue doing hard deletes, too much would break (there isn't simply a service layer we can tweak) - though through the merge functionality (otherwise very rarely through the heavily restricted "delete" button). I was going to suggest that the central change log simply not update logs of `name` to `"delete"`, but then it might cause strife for the dashboard. Perhaps omSupply should just ignore deletes for `name`? @andreievg whaddya reckon?